test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -224,6 +225,21 @@ def test_tf_serving_api_version(tensorflow_inference):
     finally:
         stop_and_remove_container(container_name, ctx)
 
+@pytest.mark.usefixtures("sagemaker_only")
+@pytest.mark.model("N/A")
+def test_sm_toolkit_and_ts_version_pytorch(pytorch_inference, region):
+    _test_sm_toolkit_and_ts_version(pytorch_inference, region)
+
+@pytest.mark.usefixtures("sagemaker_only")
+@pytest.mark.model("N/A")
+def test_sm_toolkit_and_ts_version_pytorch_graviton(pytorch_inference_graviton, region):
+    _test_sm_toolkit_and_ts_version(pytorch_inference_graviton, region)    
+
+@pytest.mark.usefixtures("sagemaker_only")
+@pytest.mark.model("N/A")
+def test_sm_toolkit_and_ts_version_pytorch_neuron(pytorch_inference_neuron, region):
```

Comment:
check with @aws-vrnatham - I am not sure if pytorch neuron images are supported on sagemaker, but this might be ok to add them if they can be used on SM

Reply:
Based on the logic in the code test below, it seems that this test should always fail - as you stated that the test only covers CPU and GPU images. Why add it if that's the case?

Reply:
fixed or no longer an issue